### DISCUSSION TITLE
create record with crmid

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14592] [How to]
I have free slot in crmid numbers of account (also in crmentity table) and need to assign one new record to certain crmid (very special case). How to manage it safely in yetiforce? Not the best in mysql though

**Comment by vovpff** [comment]
What is your reason?

**Reply by IanMuller** [reply]
account with certain crmid has been deleted, but some printed QR codes includes var with this crmid so, need to recreate account as per QR code

**Reply by vovpff** [reply]
What is the problem to generate new qr?

I think this os impossible to create new accounts with crmid of old records.

**Reply by IanMuller** [reply]
> What is the problem to generate new qr?

3000 stickers already printed. 

I think it is possible, but I am asking so no hidden implications happen. Not much idea of structure and interactions between tables

**Reply by vovpff** [reply]
I think you need to modify system core. This is crazy idea.

**Reply by mariuszkrzaczkowski** [reply]
add a new record and replace it in several tables on the basis of the database
alternative to create a record with a script
for both ideas you need a programmer who knows crm and your system
